docs/templates.md

```diff
@@ -4,3 +4,28 @@ List of actions https://backstage.broadinstitute.org/create/actions
 
 Interactive Editor for Software Templates
 https://backstage.broadinstitute.org/create/edit
+
+Local Development of Software Templates
+
+Becase we are breaking up software templates into reusable components, we have to use the `url` field in the app-config.yaml to reference the templates. This means that when you are developing a new template, you need to push your changes to a branch in order to test them. You can also create a monolithic template that includes all the components, and then split the config into separate files once you are happy with the result.
```

Comment:
Because

Reply:
oh yikes, I forgot that got checked in! :doh:

Reply:
fixed